Update managed export instructions (#858)
microsoft/finops-toolkit: docs/_reporting/hubs/configure-scopes.md

<details open markdown="1">
   <summary class="fs-2 text-uppercase">On this page</summary>

> - [🛠️ Configure exports manually](#️-configure-exports-manually)

Reviewer: Why are you making this one first? Don't we want to promote managed exports more?

Author: Just because manual exports are the simplest to setup today and I believe most of the customers are going to still be prefer this option. At least until we have figured out the custom UI to deploy hubs. 
In my opinion, manual exports +  PowerShell command to backfill data is good for most of the scenarios.

  _Microsoft Cost Management does not support managed exports for Microsoft Customer Agreement billing accounts. Please [configure Cost Management exports manually](#️-configure-exports-manually)._
</blockquote>

For the most seamless experience, we recommend allowing FinOps hubs to manage exports for you when possible. This option requires the least effort to maintain over time.

> <!-- TODO: The  sentence above can be changed when ME can be configured through UI and not command line -->

Reviewer: Is this about "simplest to setup"? If so, I think that's a fair point. But I don't think we need the comment unless you're going to comment out the sentence you want to revert back to later. Either way, I think the simpler sentence is totally fine without the "simplest to setup" part. The ultimate point is conveyed.

> 
> <!-- TODO: The  sentence above can be changed when ME can be configured through UI and not command line -->

Reviewer: Is this about "simplest to setup"? If so, I think that's a fair point. But I don't think we need the comment unless you're going to comment out the sentence you want to revert back to later. Either way, I think the simpler sentence is totally fine without the "simplest to setup" part. The ultimate point is conveyed and we probably don't need this comment.
```suggestion
```

<br>

## 🛠️ Configure exports manually

Manual exports are available if you cannot grant permissions for your scope. This method allows you to create Cost Management exports to accomplish the same goal.

> 1. [Create a new FOCUS cost export](https://learn.microsoft.com/azure/cost-management-billing/costs/tutorial-export-acm-data?tabs=azure-portal) using the following settings:

Reviewer: ```suggestion
1. [Create a new FOCUS cost export](https://learn.microsoft.com/azure/cost-management-billing/costs/tutorial-improved-exports) using the following settings:
```

>    <!-- TODO: Replace the portal link with the docs link when exports v2 docs are available. -->
> 

Reviewer: ```suggestion
```

   - **Type of data** = `Cost and usage details (FOCUS)`<sup>1</sup>
   - **Dataset version** = `1.0`<sup>2</sup>
   - **Frequency** = `Daily export of month-to-date costs`<sup>3</sup>
   - **File partitioning** = On
   - **Overwrite data** = Off<sup>4</sup>
   - **Storage account** = (Use subscription/resource deployed with your hub)
   - **Container** = `msexports`

>    - **Directory** = (Use the resource ID of the scope<sup>5</sup> you're exporting without the first "/")

Reviewer: Side note: We should revisit this. It's no longer important what the directory path is. In 0.3 and earlier, we used this to identify the scope. In 0.4 we use metadata. Given the new limitations around characters, we might want to simplify the recommendation.

Author: https://github.com/microsoft/finops-toolkit/issues/863

     - _**Department:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}`_

    <blockquote class="tip" markdown="1">
      _If you get an error about invalid characters for a billing scope, replace the unsupported characters (e.g., `:`) in the billing account ID with a dash (`-`)._
    </blockquote>

2. Create another export with the same settings except set **Frequency** to `Monthly export of last month's costs`.
3. Run your exports to initialize the dataset.
   - Exports can take up to a day to show up after first created.
   - Use the **Run now** command at the top of the Cost Management Exports page.
   - Your data should be available within 15 minutes or so, depending on how big your account is.
   - If you want to backfill data, open the export details and select the **Export selected dates** command to export one month at a time or use the [Start-FinOpsCostExport PowerShell command](../../_automation/powershell/cost/Start-FinOpsCostExport.md) to export a larger date range.
4. Repeat steps 1-3 for each scope you want to monitor.

> 

Reviewer: ```suggestion
_<sup>1) FinOps hubs 0.2 and beyond requires FOCUS cost data. As of July 2024, the option to export FOCUS cost data is only accessible from the central Cost Management experience in the Azure portal. If you do not see this option, please search for or navigate to [Cost Management Exports](https://portal.azure.com/#blade/Microsoft_Azure_CostManagement/Menu/open/exports).</sup>_
_<sup>2) FinOps hubs 0.4 supports both FOCUS 1.0 and FOCUS 1.0 preview. Power BI reports in 0.4 are aligned to FOCUS 1.0 regardless of whether data was ingested as FOCUS 1.0 preview. If you need 1.0 preview data and reports, please use FinOps hubs 0.3.</sup>_
_<sup>3) Configuring a daily export starts in the current month. If you want to backfill historical data, create a one-time export and set the start/end dates to the desired date range.</sup>_

<br>

```

## 🔐 Configure managed exports

Managed exports allow FinOps hubs to setup and maintain Cost Management exports for you. To enable managed exports, you must grant Azure Data Factory access to read data across each scope you want to monitor.

![Screenshot of the hubs supported scopes](https://raw.githubusercontent.com/microsoft/finops-toolkit/11b24a372b9bd57e7829c4224e2569647908b261/src/images/hubs-scopes.jpg)

> > **Note:** Managed exports are only available in FinOps hubs 0.4 and beyond.

Reviewer: ```suggestion
<blockquote class="note" markdown="1">
  _Managed exports are only available in FinOps hubs 0.4 and beyond._
</blockquote>
```

<br>

> _<sup>1) FinOps hubs 0.2 and beyond requires FOCUS cost data. As of July 2024, the option to export FOCUS cost data is only accessible from the central Cost Management experience in the Azure portal. If you do not see this option, please search for or navigate to [Cost Management Exports](https://portal.azure.com/#blade/Microsoft_Azure_CostManagement/Menu/open/exports).</sup>_
> _<sup>2) FinOps hubs 0.4 supports both FOCUS 1.0 and FOCUS 1.0 preview. Power BI reports in 0.4 are aligned to FOCUS 1.0 regardless of whether data was ingested as FOCUS 1.0 preview. If you need 1.0 preview data and reports, please use FinOps hubs 0.3.</sup>_
> _<sup>3) Configuring a daily export starts in the current month. If you want to backfill historical data, create a one-time export and set the start/end dates to the desired date range.</sup>_

Reviewer: These notes should move up to be under the list that was moved up.
```suggestion
```
